modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -223,6 +224,26 @@ namespace {
         openspace::properties::Property::Visibility::User
     };
 
+    // This `Renderable` visualizes field lines, mainly a sequence of time steps but work
+    // with only one time step too. A sequence is a data source consisting of multiple
+    // data files that each correspond to a specific time, and is therefor time varying
+    // like the name of the renderable suggests.
+    //
+    // `LoadingType` can be specified in two ways;
+    //
+    // `StaticLoading`: In this case all data is loaded when starting OpenSpace. A
+    // `SourceFolder` is then mandatory. The data format is also required to be set with
+    // `InputFileType`.
+    //
+    // `DynamicDownloading`: This case downloads the data during run time. For this a few
+    // parameters are required: `InfoURL`, `DataURL` and `DataID` are used to specify the
+    // data source to use.
+    //
+    // When using CDF data `SeedPointDirectory` is required. Some prior knowledge of the
+    // data is needed to use it in this way. `TracingVariable` needs to be set and
+    // `ExtraVariables` will have to match what parameters are in the CDF data. Using CDF
+    // directly in this `Renderable` is not recommended. Rather use
+    // `KameleonVolumeToFieldlinesTask` first, to save the data to .osfls or .json
```

Comment:
```suggestion
    // `KameleonVolumeToFieldlinesTask` task first, to save the data to .osfls or .json
```

Reply:
I know its double, but technically `KameleonVolumeToFieldlinesTask` is the name of the task